source/common/quic/envoy_quic_client_session.cc

```diff
@@ -252,7 +252,7 @@ void EnvoyQuicClientSession::setHttp3Options(
   }
   static_cast<EnvoyQuicClientConnection*>(connection())
       ->setNumPtosForPortMigration(PROTOBUF_GET_WRAPPED_OR_DEFAULT(
-          http3_options.quic_protocol_options(), num_timeouts_to_trigger_port_migration, 4));
+          http3_options.quic_protocol_options(), num_timeouts_to_trigger_port_migration, 0));
```

Comment:
I just realized this would turn off port migration for everyone else... I'm assuming non-mobile users aren't using it, but are we sure?  we probably need to at least have a release note about this?

Reply:
I'm not entirely sure that this code doesn't execute anywhere else. But since it's prone to UAF issues, turning it off does more good than evil.
done adding a change log.

Reply:
What are the UAF issues? Should it be forcefully disabled (unconfigurable) until that is resolved?